### DISCUSSION TITLE
RPi gadget mode

### DISCUSSION THREAD

**Discussion by wormstein** [EdNekebno/DGTCentaurMods#61] [Ideas]
Hey,
this is something I am looking over: https://howchoo.com/pi/raspberry-pi-gadget-mode

This is just an example for SSH but I wonder if we could enable just mass storage device, or maybe socket comm... Maybe enable e-board emulation over USB.

This mod will require doing a hole in the side of the board to access the USB.
Just a thought... nothing high priority.

**Comment by EdNekebno** [comment]
Yeh...mine has a hole or two already :) . e-board over USB is something I wondered about.

**Comment by wormstein** [comment]
Looks like this is not possible on DGT Centaur because Raspi has power over GPIO already. According to schematics all 5V pins on the board are on the came rail without any kind of protections. Plugging a PC on the USB port might cause bad things to happen. Because the external row of GPIO pins used, it is clear that Raspi gets a 5V. (5V pins are on that side)... No wonder DGT decided to put the charging port on the controller and never expose USBs on Raspi.
